rules/typescript.js

```diff
@@ -19,6 +19,11 @@ module.exports = {
     '@typescript-eslint/no-useless-constructor': 'error',
     '@typescript-eslint/prefer-for-of': 'error',
     '@typescript-eslint/prefer-optional-chain': 'error',
+    "@typescript-eslint/consistent-type-imports": "error",
+    "@typescript-eslint/explicit-module-boundary-types": "off",
+    "@typescript-eslint/ban-ts-comment": "off",
+    "@typescript-eslint/camelcase": "off",
+    "@typescript-eslint/no-var-requires": "off",
```

Comment:
As for the `no-var-requires` rule I believe until we have projects using common js this rule doesn't make sense. or?

Reply:
In the LP there are some places where we use `require`, so I've disabled this rule

Reply:
I would generally use TS as a type checker, not as a JS flavour with its own opinions about how imports work.

Reply:
I will also add it as a topic for the weekly.